erpnext/accounts/report/deferred_revenue_and_expense/test_deferred_revenue_and_expense.py

```diff
@@ -10,16 +11,15 @@
 from erpnext.accounts.report.deferred_revenue_and_expense.deferred_revenue_and_expense import (
 	Deferred_Revenue_and_Expense_Report,
 )
+from erpnext.accounts.test.accounts_mixin import AccountsTestMixin
 from erpnext.accounts.utils import get_fiscal_year
 from erpnext.buying.doctype.supplier.test_supplier import create_supplier
 from erpnext.stock.doctype.item.test_item import create_item
 
 
-class TestDeferredRevenueAndExpense(unittest.TestCase):
+class TestDeferredRevenueAndExpense(FrappeTestCase, AccountsTestMixin):
```

Comment:
Just from the MRO point of view, the order of inheritance should be flipped. 


Anyway, you're not overriding anything right now so doesn't matter much. Nice work on removing duplication in tests :+1: